lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -475,11 +475,7 @@ public function & getEntityChangeSet($entity)
         $oid  = spl_object_id($entity);
         $data = [];
 
-        if (!isset($this->entityChangeSets[$oid])) {
-            return $data;
-        }
-
-        return $this->entityChangeSets[$oid];
+        return $this->entityChangeSets[$oid] ?? $data;
```

Comment:
This change is incorrect: the method returns a reference, so an expression is not valid when combined with `return`

Reply:
Should I rollback so?

Reply:
Yeap